client/components/jetpack/jetpack-product-info/regular-list.tsx

```diff
@@ -1,14 +1,55 @@
-import { FunctionComponent, ReactNode } from 'react';
+import { useRef, useEffect, type FC, type ReactNode } from 'react';
```

Comment:
Nit: feel free to import types in a separate `import type` import, mostly for better readability.